db/migrate/20241107134140_add_expires_at_to_access_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddExpiresAtToAccessTokens < ActiveRecord::Migration[6.1]
+  def change
+    add_column :access_tokens, :expires_at, :datetime
```

Comment:
usually we add some options for postgres and not sure others when doing DDL, is this not necessary here? 

```
disable_ddl_transaction! if System::Database.postgres?
```

Reply:
Everything else looks nice, thank you!

Reply:
I don't know if this is required or not, but here is it: https://github.com/3scale/porta/pull/3939/commits/5437e35f957ac018d69f0dcdca6af6f7ed36c87f

Reply:
Thanks, just we need to test how long will this migration take in production and do manually with a maintenance window if it takes long.

Reply:
I don't want to think about it. Needs to be just tried out.

Reply:
Don't you prefer to just deploy directly to production on a Friday afternoon? you're not cool anymore...

Reply:
I'm not sure I was ever cool.. ever since I updated replaced glibc on the dial-up slack server of an Internet provider.